modules/file/file.go

```diff
@@ -44,14 +46,52 @@ func debugprint(format string, a ...interface{}) {
 }
 
 type module struct {
+	SandboxProfile sandbox.SandboxProfile
 }
 
 func (m *module) NewRun() modules.Runner {
 	return new(run)
 }
 
+func (m *module) GetSandboxProfile() sandbox.SandboxProfile {
+	return m.SandboxProfile
+}
+
 func init() {
-	modules.Register("file", new(module))
+	m := new(module)
+	sandbox := sandbox.SandboxProfile{
+		DefaultPolicy: seccomp.ActTrap,
+		Filters: []sandbox.FilterOperation{
+			sandbox.FilterOperation{
+				FilterOn: []string{
+					"pread64",
+					"lstat",
+					"close",
+					"openat",
+					"stat",
+					"readlinkat",
+					"write",
+					"mmap",  // GO
```

Comment:
nitpick: would clarify "//GO" into "required by Go runtime" or something like that